script.js

```diff
@@ -1,74 +1,84 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable class-methods-use-this */
+
 let booklist = [];
 const submitBtn = document.getElementById('submit-btn');
 const form = document.getElementById('form');
 const bookTitle = document.getElementById('book-title');
 const author = document.getElementById('author');
 
-// Storage
-
-function updateStorage(data) {
-  localStorage.setItem('bookLists', JSON.stringify(data));
-}
+class Book {
+  updateStorage(data) {
+    localStorage.setItem('bookLists', JSON.stringify(data));
+  }
 
-// Remove title function
+  // Remove title function
 
-function removeBook(title) {
-  booklist = booklist.filter((book) => book.title !== title);
-  updateStorage(booklist);
-}
+  removeBook(title) {
+    const rm = new Book();
+    booklist = booklist.filter((book) => book.title !== title);
+    rm.updateStorage(booklist);
+  }
```

Comment:
- [ ] Kindly implement this deletion function with a different parameter, like ID. When I add two books of the same title and delete one, it deletes the other one as well. Kindly fix that.